coalib/coala_delete_orig.py

```diff
@@ -9,11 +9,24 @@
 
```

Comment:
commit message:
1. You don't create a helper function, you modularized functionality.
2. Created -> Create or Creates (present tense 😄)
3. the helper function produces errors? That's bad :O I think you meant "avoid"